atd-vze/src/views/Crashes/crashGridTableParameters.js

```diff
@@ -66,6 +66,60 @@ export const crashGridTableColumns = {
   },
 };
 
+export const nonCR3CrashGridTableColumns = {
+  form_id: {
+    primary_key: false, // We say no here bc there is no page to link to
+    searchable: true,
+    sortable: true,
+    label_search: "Search by Crash Form ID",
+    label_table: "Crash Form ID",
+    type: "Int",
+  },
+  date: {
+    primary_key: false,
+    searchable: true,
```

Comment:
I think `searchable` needs to be `false` here. This is creating another search field in the `GridTable` that is blank and can break the query if selected.

<img width="878" alt="Screen Shot 2020-01-05 at 10 23 05 AM" src="https://user-images.githubusercontent.com/37249039/71782947-968aa780-2fa5-11ea-9014-4beb40d0d343.png">

Reply:
Yep, line 80 to be specific seems to cause since there is no label defined.

I tested by turning the line to `false`, also tested to make sure the date ranges work fine when searching with that false value. It all seems to work fine:

![2020-01-05_18-31-21](https://user-images.githubusercontent.com/5282430/71788631-b774ec00-2fe9-11ea-8530-7e7601659c00.png)
![2020-01-05_18-30-36](https://user-images.githubusercontent.com/5282430/71788632-b774ec00-2fe9-11ea-8e1a-8725a8f73a4c.png)
![2020-01-05_18-30-21](https://user-images.githubusercontent.com/5282430/71788633-b774ec00-2fe9-11ea-9e33-d9b617c90524.png)